COMMERCE-10201 Hover on plus icon button for Options is not correctly translated

elocamp opened the pull request:
When a label was being set, there was not a treatment on the key language.
Purposed fix was to add this treatment when set a label.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

brianikim commented:
Hello @matthewchan123 ,

I will have to close this PR. You are unfortunately missing a period here: https://github.com/brianikim/liferay-portal/blob/861f9856ccf6b67b01d0d3abeef020fc263cb259/modules/apps/connected-app/connected-app-web/src/main/resources/content/Language.properties#L4

Please regenerate the buildLang as well.

Sincerely,
Brian Kim

matthewchan123 commented:
ok, no problem. I had been deliberating whether to add the period or not, because the ticket creator had not added one in his request.

JanisZhang committed:
LPS-165338 Remove unused Reference field _ddmStructureService,as its usages was removed in 8ba6c233960d8d46dbab63e096937debbd7c3cb5.

JanisZhang committed:
LPS-165338 Remove unused Reference field _jsonFactory, its usages was removed in 48120813fc1944d0fe53e0672ecd909abdf4c240.

JanisZhang committed:
LPS-165338 Remove unused Reference field _ddmFormFieldTypeServicesTracker, its usages was removed in 4de6ae1e80dbb17f8571d0a955dd6ee55d7cd9b2.

JanisZhang committed:
LPS-165338 Remove unused Reference field _ddmDataProviderInstanceService, its usages was removed in 80c91060737dca8eb6882f39520c38164f5ff934.

JanisZhang committed:
LPS-165338 Make JSONFactory field used; although the direct usages was removed in af07059a739eae579e9e4f2bcb7900101c1c4cb3, the JSONFactoryUtil usage by the class it creates and uses should still be protected by the reference.

JanisZhang committed:
LPS-165338 Remove unused Reference field _jsonDDMFormDeserializer, its usages was removed in 5a93ea812ae6e329d2078d510a80f0aef3ff6de8.

JanisZhang committed:
LPS-165338 Remove unused Reference field _ddmFormInstanceLocalService, as it already been referenced in DDMFormInstanceUADDisplay's parent class.

slnn committed:
LPS-166048 Remove reference field _userLocalService from JournalArticleContentDashboardItemFactory, FileEntryContentDashboardItemFactory and BlogsEntryContentDashboardItemFactory

Note:
Field _userLocalService in JournalArticleContentDashboardItemFactory was added to the class and applied usage by LPS-117280:eb7109e8f6ad3c84ddc1e723eed36518685c80cf, its usage was removed by LPS-119190:f5ac553d55984c9e8994837779bb5b29973f9b09.

Field _userLocalService in FileEntryContentDashboardItemFactory and BlogsEntryContentDashboardItemFactory was added to these classes when creating these classes but no one using it

slnn committed:
LPS-166048 Remove reference fields _sharingConfigurationFactory and _sharingPermission from class ContentDashboardAdminPortlet since no one uses them directly in the class

slnn committed:
LPS-166048 Remove reduant reference field _classNameLocalService from class ContentDashboardAdminPortlet since the field is not used directly in the class and the class's another field reference _contentDashboardItemFactoryTracker has declared a field reference for ClassNameLocalService

slnn committed:
LPS-166048 Remove reference field _assetVocabularyLocalService from UpdateContentDashboardConfigurationMVCActionCommand since it is added and applyied usage by LPS-128990:7a5674d57504739eb8c98d5ab52504c602fb7e03 but usage was removed by LPS-128990:24bf462165aca405436a4a604d354cf9865ef45f

JanisZhang committed:
LPS-166022 Remove unused reference field _journalContent, its usage was removed in 2e51e57644e579e95cf55b7d80f5f89a50125fff

JanisZhang committed:
LPS-166022 Remove unused reference field _infoItemServiceTracker, it was added in 2eaaeb6a0d3ed0010e003de87594d54f2bdd2028 but never used

JanisZhang committed:
LPS-166022 Remove unused reference field _fieldsToDDMFormValuesConverter, its usage was removed in 7a4d5a6cf6472d8eb7e1bb4a772f6db6e822149b

JanisZhang committed:
LPS-166022 Remove unused reference fields, they have been referenced in JournalHelperImpl

JanisZhang committed:
LPS-166022 Remove unused reference field _groupService, it was added in 2700f809d4872e3c446d8580d09e479fe41b38e6 but never used

JanisZhang committed:
LPS-166022 Remove unused reference field _groupLocalService, its usage was removed in bbfeb4e51ce31ac61a8483fd8a7f9f9f041804dc

JanisZhang committed:
LPS-166022 Remove unused reference field _journalFolderLocalService, it was added since class creation but never used

JanisZhang committed:
LPS-166022 Remove unused reference field _portal, it has been referenced in DDMStructureLocalServiceImpl .(through _journalArticleService.deleteArticleDefaultValues() ->  _ddmStructureService.getStructure())

JanisZhang committed:
LPS-166022 Remove unused reference field _jsonFactory, it has been referenced in DDMStructureLocalServiceImpl

JanisZhang committed:
LPS-166022 Remove unused reference field _ddmStructureLocalService, its usage has been removed in 4cec2cd8ad58180705984469cb3b7b0aba6903de

JanisZhang committed:
LPS-166022 Remove unused reference field _portal, it has been referenced in JournalHelperImpl

Commit:
LPS-166034 Remove unused reference variable _portal(It was added when the class was created, but never used(eca1b86c3c0e453421313e9f2ea449c62a71ad53))

Commit:
LPS-166034 Remove unused reference variable _portal(usge was removed in task LPS-89995(c5b3fe5ddf527dcb5a3e781a437dba9345230c06))

ces-sontruong committed:
LPS-165991 Change parameter order

Commit:
COMMERCE-8863 Using GetterUtil to account for null values

Commit:
LPS-165677 HR_74 Add aria-label to clay:dropdown-actions taglib call

Commit:
LPS-165701 HR_139 Add title prop to ActionControls' dropdown trigger

slnn committed:
LPS-166158 Remove reference field _companyService from PortalInstancesLocalServiceImpl which added to the class when creating class but no directly usage, no tests failed if remove it

ambrinchaudhary committed:
LPS-166066 Add custom error message

ambrinchaudhary committed:
LPS-166066 Add key

ambrinchaudhary committed:
LPS-166066 buildLang

ealonso committed:
LPS-166142 Adds new taglib that includes the cancel and submit buttons in the correct order and with the correct styles to make those accessible, also avoid developers mistakes

ealonso committed:
LPS-166142 Uses new taglib

ozysouza committed:
LPS-164770 Add test case to Form Components should not be available on display page template

ozysouza committed:
LPS-164770 Add testcase to user can map a Form Container on masters page template

ozysouza committed:
LPS-164770 Add testcase to Form Components should available on Masters Page Template for user with custom permissions

brianchandotcom committed:
LPS-165486 SF

brianchandotcom committed:
artifact:ignore commerce-service 11.0.104 change log

brianchandotcom committed:
artifact:ignore commerce-service 11.0.104 prep next

brianchandotcom committed:
artifact:ignore commerce-service 11.0.104 artifact properties

brianchandotcom committed:
artifact:ignore commerce-taglib 5.0.18 change log

brianchandotcom committed:
artifact:ignore commerce-taglib 5.0.18 prep next

brianchandotcom committed:
artifact:ignore commerce-taglib 5.0.18 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-admin-catalog-impl 4.0.67 change log

brianchandotcom committed:
artifact:ignore headless-commerce-admin-catalog-impl 4.0.67 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-admin-catalog-impl 4.0.67 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-admin-pricing-impl 4.0.48 change log

brianchandotcom committed:
artifact:ignore headless-commerce-admin-pricing-impl 4.0.48 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-admin-pricing-impl 4.0.48 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-delivery-cart-impl 4.0.58 change log

brianchandotcom committed:
artifact:ignore headless-commerce-delivery-cart-impl 4.0.58 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-delivery-cart-impl 4.0.58 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-delivery-order-impl 1.0.9 change log

brianchandotcom committed:
artifact:ignore headless-commerce-delivery-order-impl 1.0.9 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-delivery-order-impl 1.0.9 prep next

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-account-item-selector-web 4.0.18 change log

brianchandotcom committed:
artifact:ignore commerce-account-item-selector-web 4.0.18 artifact properties

brianchandotcom committed:
artifact:ignore commerce-account-item-selector-web 4.0.18 prep next

brianchandotcom committed:
artifact:ignore commerce-account-web 4.0.25 change log

brianchandotcom committed:
artifact:ignore commerce-account-web 4.0.25 artifact properties

brianchandotcom committed:
artifact:ignore commerce-account-web 4.0.25 prep next

brianchandotcom committed:
artifact:ignore commerce-address-web 4.0.36 change log

brianchandotcom committed:
artifact:ignore commerce-address-web 4.0.36 artifact properties

brianchandotcom committed:
artifact:ignore commerce-address-web 4.0.36 prep next

brianchandotcom committed:
artifact:ignore commerce-cart-taglib 5.0.20 change log

brianchandotcom committed:
artifact:ignore commerce-cart-taglib 5.0.20 prep next

brianchandotcom committed:
artifact:ignore commerce-cart-taglib 5.0.20 artifact properties

brianchandotcom committed:
artifact:ignore commerce-catalog-web 4.0.28 change log

brianchandotcom committed:
artifact:ignore commerce-catalog-web 4.0.28 prep next

brianchandotcom committed:
artifact:ignore commerce-catalog-web 4.0.28 artifact properties

brianchandotcom committed:
artifact:ignore commerce-channel-web 4.0.46 change log

brianchandotcom committed:
artifact:ignore commerce-channel-web 4.0.46 prep next

brianchandotcom committed:
artifact:ignore commerce-channel-web 4.0.46 artifact properties

brianchandotcom committed:
artifact:ignore commerce-currency-web 4.0.26 change log

brianchandotcom committed:
artifact:ignore commerce-currency-web 4.0.26 prep next

brianchandotcom committed:
artifact:ignore commerce-currency-web 4.0.26 artifact properties

brianchandotcom committed:
artifact:ignore commerce-item-selector-web 4.0.21 change log

brianchandotcom committed:
artifact:ignore commerce-item-selector-web 4.0.21 prep next

brianchandotcom committed:
artifact:ignore commerce-item-selector-web 4.0.21 artifact properties

brianchandotcom committed:
artifact:ignore commerce-notification-web 4.0.24 change log

brianchandotcom committed:
artifact:ignore commerce-notification-web 4.0.24 artifact properties

brianchandotcom committed:
artifact:ignore commerce-notification-web 4.0.24 prep next

brianchandotcom committed:
artifact:ignore commerce-order-content-web 4.0.72 change log

brianchandotcom committed:
artifact:ignore commerce-order-content-web 4.0.72 artifact properties

brianchandotcom committed:
artifact:ignore commerce-order-content-web 4.0.72 prep next

brianchandotcom committed:
artifact:ignore commerce-order-rule-web 1.0.22 change log

brianchandotcom committed:
artifact:ignore commerce-order-rule-web 1.0.22 prep next

brianchandotcom committed:
artifact:ignore commerce-order-rule-web 1.0.22 artifact properties

brianchandotcom committed:
artifact:ignore commerce-order-web 5.0.60 change log

brianchandotcom committed:
artifact:ignore commerce-order-web 5.0.60 artifact properties

brianchandotcom committed:
artifact:ignore commerce-order-web 5.0.60 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-method-authorize-net 4.0.20 change log

brianchandotcom committed:
artifact:ignore commerce-payment-method-authorize-net 4.0.20 artifact properties

brianchandotcom committed:
artifact:ignore commerce-payment-method-authorize-net 4.0.20 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-method-mercanet 4.0.21 change log

brianchandotcom committed:
artifact:ignore commerce-payment-method-mercanet 4.0.21 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-method-mercanet 4.0.21 artifact properties

brianchandotcom committed:
artifact:ignore commerce-payment-method-money-order 4.0.14 change log

brianchandotcom committed:
artifact:ignore commerce-payment-method-money-order 4.0.14 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-method-money-order 4.0.14 artifact properties

brianchandotcom committed:
artifact:ignore commerce-payment-method-paypal 4.0.19 change log

brianchandotcom committed:
artifact:ignore commerce-payment-method-paypal 4.0.19 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-method-paypal 4.0.19 artifact properties

brianchandotcom committed:
artifact:ignore commerce-payment-web 4.0.35 change log

brianchandotcom committed:
artifact:ignore commerce-payment-web 4.0.35 prep next

brianchandotcom committed:
artifact:ignore commerce-payment-web 4.0.35 artifact properties

brianchandotcom committed:
artifact:ignore commerce-pricing-web 4.0.54 change log

brianchandotcom committed:
artifact:ignore commerce-pricing-web 4.0.54 prep next

brianchandotcom committed:
artifact:ignore commerce-pricing-web 4.0.54 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-definitions-web 7.0.72 change log

brianchandotcom committed:
artifact:ignore commerce-product-definitions-web 7.0.72 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-definitions-web 7.0.72 prep next

brianchandotcom committed:
artifact:ignore commerce-product-options-web 4.0.40 change log

brianchandotcom committed:
artifact:ignore commerce-product-options-web 4.0.40 prep next

brianchandotcom committed:
artifact:ignore commerce-product-options-web 4.0.40 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shipment-web 4.0.34 change log

brianchandotcom committed:
artifact:ignore commerce-shipment-web 4.0.34 prep next

brianchandotcom committed:
artifact:ignore commerce-shipment-web 4.0.34 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shipping-web 6.0.24 change log

brianchandotcom committed:
artifact:ignore commerce-shipping-web 6.0.24 prep next

brianchandotcom committed:
artifact:ignore commerce-shipping-web 6.0.24 artifact properties

brianchandotcom committed:
artifact:ignore commerce-tax-engine-fixed-web 4.0.28 change log

brianchandotcom committed:
artifact:ignore commerce-tax-engine-fixed-web 4.0.28 artifact properties

brianchandotcom committed:
artifact:ignore commerce-tax-engine-fixed-web 4.0.28 prep next

brianchandotcom committed:
artifact:ignore commerce-tax-web 4.0.18 change log

brianchandotcom committed:
artifact:ignore commerce-tax-web 4.0.18 prep next

brianchandotcom committed:
artifact:ignore commerce-tax-web 4.0.18 artifact properties

brianchandotcom committed:
artifact:ignore commerce-term-web 1.0.22 change log

brianchandotcom committed:
artifact:ignore commerce-term-web 1.0.22 prep next

brianchandotcom committed:
artifact:ignore commerce-term-web 1.0.22 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-speedwell-site-initializer 4.0.20 change log

brianchandotcom committed:
artifact:ignore commerce-theme-speedwell-site-initializer 4.0.20 prep next

brianchandotcom committed:
artifact:ignore commerce-theme-speedwell-site-initializer 4.0.20 artifact properties

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.64 change log

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.64 prep next

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.64 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-cart-content-web 4.0.34 change log

brianchandotcom committed:
artifact:ignore commerce-cart-content-web 4.0.34 prep next

brianchandotcom committed:
artifact:ignore commerce-cart-content-web 4.0.34 artifact properties

brianchandotcom committed:
artifact:ignore commerce-checkout-web 5.0.56 change log

brianchandotcom committed:
artifact:ignore commerce-checkout-web 5.0.56 artifact properties

brianchandotcom committed:
artifact:ignore commerce-checkout-web 5.0.56 prep next

brianchandotcom committed:
artifact:ignore commerce-inventory-web 4.0.32 change log

brianchandotcom committed:
artifact:ignore commerce-inventory-web 4.0.32 prep next

brianchandotcom committed:
artifact:ignore commerce-inventory-web 4.0.32 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-asset-categories-web 4.0.37 change log

brianchandotcom committed:
artifact:ignore commerce-product-asset-categories-web 4.0.37 prep next

brianchandotcom committed:
artifact:ignore commerce-product-asset-categories-web 4.0.37 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-content-web 4.0.67 change log

brianchandotcom committed:
artifact:ignore commerce-product-content-web 4.0.67 prep next

brianchandotcom committed:
artifact:ignore commerce-product-content-web 4.0.67 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-ddm-impl 4.0.16 change log

brianchandotcom committed:
artifact:ignore commerce-product-ddm-impl 4.0.16 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-ddm-impl 4.0.16 prep next

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.25 change log

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.25 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.25 prep next

brianchandotcom committed:
artifact:ignore commerce-product-tax-category-web 4.0.28 change log

brianchandotcom committed:
artifact:ignore commerce-product-tax-category-web 4.0.28 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-tax-category-web 4.0.28 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 4.0.36 change log

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 4.0.36 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 4.0.36 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-type-virtual-web 4.0.32 change log

brianchandotcom committed:
artifact:ignore commerce-product-type-virtual-web 4.0.32 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-virtual-web 4.0.32 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fedex 5.0.23 change log

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fedex 5.0.23 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fedex 5.0.23 prep next

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fixed-web 4.0.35 change log

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fixed-web 4.0.35 prep next

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fixed-web 4.0.35 artifact properties

brianchandotcom committed:
artifact:ignore commerce-subscription-web 4.0.32 change log

brianchandotcom committed:
artifact:ignore commerce-subscription-web 4.0.32 prep next

brianchandotcom committed:
artifact:ignore commerce-subscription-web 4.0.32 artifact properties

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.34 change log

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.34 prep next

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.34 artifact properties

brianchandotcom committed:
artifact:ignore commerce-wish-list-web 4.0.30 change log

brianchandotcom committed:
artifact:ignore commerce-wish-list-web 4.0.30 prep next

brianchandotcom committed:
artifact:ignore commerce-wish-list-web 4.0.30 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-minium-impl 5.0.19 change log

brianchandotcom committed:
artifact:ignore commerce-theme-minium-impl 5.0.19 prep next

brianchandotcom committed:
artifact:ignore commerce-theme-minium-impl 5.0.19 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-minium-site-initializer 5.0.27 change log

brianchandotcom committed:
artifact:ignore commerce-theme-minium-site-initializer 5.0.27 prep next

brianchandotcom committed:
artifact:ignore commerce-theme-minium-site-initializer 5.0.27 artifact properties

brianchandotcom committed:
build.gradle auto SF

Commit:
LPS-166122 ComparisonFailure in ObjectEntryLocalServiceTest.testUpdateObjectEntry

brianchandotcom committed:
artifact:ignore analytics-reports-blogs-impl 1.0.2 change log

brianchandotcom committed:
artifact:ignore analytics-reports-blogs-impl 1.0.2 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-blogs-impl 1.0.2 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-impl 2.0.6 change log

brianchandotcom committed:
artifact:ignore analytics-reports-impl 2.0.6 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-impl 2.0.6 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-layout-display-page-impl 1.0.11 change log

brianchandotcom committed:
artifact:ignore analytics-reports-layout-display-page-impl 1.0.11 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-layout-display-page-impl 1.0.11 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-layout-impl 1.0.10 change log

brianchandotcom committed:
artifact:ignore analytics-reports-layout-impl 1.0.10 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-layout-impl 1.0.10 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.62 change log

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.62 prep next

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.62 artifact properties

brianchandotcom committed:
artifact:ignore commerce-machine-learning-impl 5.0.32 change log

brianchandotcom committed:
artifact:ignore commerce-machine-learning-impl 5.0.32 prep next

brianchandotcom committed:
artifact:ignore commerce-machine-learning-impl 5.0.32 artifact properties

brianchandotcom committed:
artifact:ignore frontend-theme-dxp-override 4.0.35 artifact properties

brianchandotcom committed:
artifact:ignore frontend-theme-dxp-override 4.0.35 prep next

brianchandotcom committed:
artifact:ignore portal-search-similar-results-web 3.0.28 change log

brianchandotcom committed:
artifact:ignore portal-search-similar-results-web 3.0.28 prep next

brianchandotcom committed:
artifact:ignore portal-search-similar-results-web 3.0.28 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-web 5.0.58 change log

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-web 5.0.58 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-web 5.0.58 artifact properties

brianchandotcom committed:
artifact:ignore saml-addon-keep-alive-web 5.0.19 change log

brianchandotcom committed:
artifact:ignore saml-addon-keep-alive-web 5.0.19 prep next

brianchandotcom committed:
artifact:ignore saml-addon-keep-alive-web 5.0.19 artifact properties

brianchandotcom committed:
artifact:ignore saml-opensaml-integration 5.3.38 change log

brianchandotcom committed:
artifact:ignore saml-opensaml-integration 5.3.38 prep next

brianchandotcom committed:
artifact:ignore saml-opensaml-integration 5.3.38 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-service 3.0.20 change log

brianchandotcom committed:
artifact:ignore search-experiences-service 3.0.20 prep next

brianchandotcom committed:
artifact:ignore search-experiences-service 3.0.20 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.19 change log

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.19 prep next

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.19 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.36 change log

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.36 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.36 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-impl 4.0.27 change log

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-impl 4.0.27 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-impl 4.0.27 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore saml-web 5.0.41 change log

brianchandotcom committed:
artifact:ignore saml-web 5.0.41 prep next

brianchandotcom committed:
artifact:ignore saml-web 5.0.41 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
7.4.3.48 GA48 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bookmarks, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream

brianchandotcom committed:
7.4.13 Update 48 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bookmarks, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream, antivirus, enterprise-product-notification, multi-factor-authentication, portal-search-elasticsearch-cross-cluster-replication, portal-search-elasticsearch-monitoring, portal-search-learning-to-rank, portal-search-similar-results, portal-search-tuning, saml, search-experiences

brianchandotcom committed:
LPS-166066 Wordsmith

brianchandotcom committed:
LPS-166066 Regen

liferay-translation committed:
LPS-77699 Update Translations

carolmariaabb committed:
LPS-165991 Auto SF

brianchandotcom committed:
Revert "LPS-165486 SF"

This reverts commit 6def0503a33e73bc1db099e46032140201da5c5f.

brianchandotcom committed:
COMMERCE-8863 SF

Commit:
COMMERCE-10201 - add-option language key has been treated

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
